server/integration/endpoint/src/main/resources/subsystem-templates/infinispan-endpoint.xml

```diff
@@ -135,6 +131,24 @@
          </router-connector>
       </replacement>
    </supplement>
+   <supplement name="memcached-local">
+      <replacement placeholder="@@default-cache-container@@" attributeValue="local" />
+      <replacement placeholder="CONNECTORS">
+         <memcached-connector socket-binding="memcached" cache-container="local" cache="default"/>
```

Comment:
I thought memcached required "memcachedCache". Is this limitation fixed already?